src/terrain/generation/land_generator.hpp

```diff
@@ -39,6 +39,83 @@ namespace terrain {
 
 namespace generation {
 
+class JsonToTile {
```

Comment:
Needs a virtual dtor

And put all of these in a `namespace stampers` or something